contracts/utils/players_registry.cairo

```diff
@@ -1,37 +1,44 @@
-######### Players registry
-## A contract to record all addresses who participated, and which exercises and workshops they completed
+# ######## Players registry
+# # A contract to record all addresses who participated, and which exercises and workshops they completed
 
 %lang starknet
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.uint256 import (
-    Uint256, uint256_add, uint256_sub, uint256_le, uint256_lt, uint256_check
+    Uint256,
+    uint256_add,
+    uint256_sub,
+    uint256_le,
+    uint256_lt,
+    uint256_check,
 )
 from starkware.cairo.common.math import assert_not_zero
-from starkware.starknet.common.syscalls import (get_caller_address)
+from starkware.starknet.common.syscalls import get_caller_address
 #
 # Declaring storage vars
 # Storage vars are by default not visible through the ABI. They are similar to "private" variables in Solidity
 #
 
 @storage_var
-func has_validated_exercice_storage(account: felt, workshop: felt, exercise: felt) -> (has_validated_exercice_storage: felt):
+func has_validated_exercice_storage(account : felt, workshop : felt, exercise : felt) -> (
+    has_validated_exercice_storage : felt
+):
 end
 
 @storage_var
-func exercises_and_admins_accounts(account: felt) -> (permission: felt):
+func exercises_and_admins_accounts(account : felt) -> (permission : felt):
 end
 
 @storage_var
-func next_player_rank_storage() -> (next_player_rank_storage: felt):
+func next_player_rank_storage() -> (next_player_rank_storage : felt):
 end
 
 @storage_var
-func players_registry_storage(rank: felt) -> (account: felt):
+func players_registry_storage(rank : felt) -> (account : felt):
 end
 
 @storage_var
-func players_ranks_storage(account: felt) -> (rank: felt):
+func players_ranks_storage(account : felt) -> (rank : felt):
```

Comment:
nit:

Since storage variables are annotated by @storage_variable we can remove _storage from the variable name